novawallet/Modules/Transfer/Operation/AssetTransferAggregationFactory.swift

```diff
@@ -59,9 +59,9 @@ final class AssetTransferAggregationFactory: AssetTransferAggregationFactoryProt
     }
 
     func createCanPayFeeWrapper(in chainAsset: ChainAsset) -> CompoundOperationWrapper<Bool> {
-        if chainAsset.chain.hasSwapHub {
+        if chainAsset.chain.hasAssetHubTransferFees {
             return createAssetHubCanPayFee(for: chainAsset)
-        } else if chainAsset.chain.hasSwapHydra {
+        } else if chainAsset.chain.hasHydationTransferFees {
```

Comment:
These changes were made previously but were lost during a bit of code restructuring. Fortunately, the manually tested chain has both `hasSwapHub` and `hasAssetHubTransferFees`, so it was still valid. However, I retested it to ensure that it works as before